repository/src/main/java/org/alfresco/repo/event2/EventSender.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * #%L
+ * Alfresco Repository
+ * %%
+ * Copyright (C) 2005 - 2023 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.repo.event2;
+
+import java.util.Optional;
+import java.util.concurrent.Callable;
+
+import org.alfresco.repo.event.v1.model.RepoEvent;
+
+public interface EventSender
+{
+    void accept(Callable<Optional<RepoEvent<?>>> eventProducer);
```

Comment:
JavaDoc is required. Since this is an interface, a JavaDoc for the interface/method would make it a lot easier for implementations.

Reply:
I forgot about this interface - JavaDoc added